zookeeper-client/zookeeper-client-c/src/cli.c

```diff
@@ -678,15 +678,15 @@ int main(int argc, char **argv) {
     }
     if (argc > 2) {
       if(strncmp("cmd:",argv[2],4)==0){
```

Comment:
Possible to extract this out into a method and add a test for it?

Reply:
Sure.
Honestly I don't know how tests are implement in C projects.
I will take a look at existing tests and try to come back with a test.

Is there any way to run a single C test?

Btw I will check existing docs

Reply:
> Is there any way to run a single C test?

Not that I'm aware of - iirc in the past I've commented out from the makefile (does seem like a good addition though...)